### DISCUSSION TITLE
Express Search has redundant titles and counts

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5653] [Feature Requests]
<img width="565" alt="image" src="https://user-images.githubusercontent.com/37256050/192672697-e9297919-25bb-49cb-97ba-0448430c6ef5.png">

> The second heading is redundant now in my opinion. If you agree, I can open another issue for that. If you see a purpose, we can leave it as-is.

_Originally posted by @grantfitzsimmons in https://github.com/specify/specify7/issues/1681#issuecomment-1260303047_

**Comment by CarolineDenis** [comment]
Seems redundant to me

**Comment by maxpatiiuk** [comment]
They are there because removing them is harder than it might seem, where as keeping them is no big issue

Removing is hard because it's rendered by separate components - the component that renders the orange bar needs to have that text as it's required for the handle.

The solution might be to add a prop to <QueryResults> to tell it to omit displaying the title, but it already has a ton of props, so I didn't want to add one more for something this minor

**Comment by CarolineDenis** [comment]
@maxpatiiuk could we close this issue?

**Comment by maxpatiiuk** [comment]
Don't know.
We need to significantly improve the express search page at some point, and this could be one of the improvements
